packages/analytics-browser/src/config/joined-config.ts

```diff
@@ -31,6 +32,8 @@ export class BrowserJoinedConfigGenerator {
       this.config.defaultTracking = remoteConfig.defaultTracking;
     }
     this.config.loggerProvider.debug('Joined configuration: ', JSON.stringify(remoteConfig, null, 2));
+    this.config.requestMetadata ??= new RequestMetadata();
+    this.config.requestMetadata.sdk.metrics.histogram.remote_config_fetch_time = this.remoteConfigFetch?.fetchTime;
```

Comment:
NIT: Can this move inside the request metadata class? Something like
```
requestMetadata.recordHistogram(key, value)
```

Reply:
The first line checks if `this.config.requestMetadata` is undefined. If it is, then create a new empty RequestMetadata. And the second line only sets fetch time. 

I'd like to keep how it's until a second key value pair is added. wdyt?

Reply:
Not a blocker - its a NIT: This is mostly to keep this class readable. This line can be a little overwhelming to read

```    
this.config.requestMetadata.sdk.metrics.histogram.remote_config_fetch_time = this.remoteConfigFetch?.fetchTime;
```

When reading this class, all we want to know that we are tracking the fetch time. How that is being tracked and where it is being stored is not something the config class cares about.

Having a function like

```
requestMetadata.recordHistogram(key, value)
```

just makes this method more readable. 

The RequestMetadata class cares about how that data is populated and where it needs to be stored.

Reply:
I start to resonate with it that `recordHistogram()` is more readable and elegant I added it at https://github.com/amplitude/Amplitude-TypeScript/pull/772/commits/46b3c20d0149d96dfc51649881f4b178dd424fab